app/recordtransfer/views/post_submission.py

```diff
@@ -29,21 +36,22 @@ class SubmissionDetail(UserPassesTestMixin, DetailView):
     template_name = "recordtransfer/submission_detail.html"
     context_object_name = "submission"
 
-    def get_object(self, queryset=None) -> Submission:
+    def get_object(self, queryset: Any = None) -> Submission:
```

Comment:
I think you can be more specific here:

```suggestion
    def get_object(self, queryset: Optional[QuerySet] = None) -> Submission:
```